certification/internal/shell/has_required_labels.go

```diff
@@ -38,10 +38,10 @@ func (p *HasRequiredLabelsCheck) validate(labels map[string]string) (bool, error
 
 	if len(missingLabels) > 0 {
 		log.Warn("expected labels are missing:", missingLabels)
-		return false, nil
 	}
 
-	return true, nil
+	log.Warn("expected labels are missing:", missingLabels)
```

Comment:
Thank you Emily. Now that you are touching this file, could you please address the following improvements?
1) Please define requiredLabels as a const.
2) Are we printing the warning message twice? I would like to keep the previous logic where we check for a condition before printing it.